### DISCUSSION TITLE
Are there any plans to integreate multiple PMS feature?

### DISCUSSION THREAD

**Discussion by xinput** [croneter/PlexKodiConnect#1280] [Q&A] [answered]
Hello, I just wanted to ask if there are any plans to add a feature to add multiple PMS sources to the sync.
I'm currently having 2 servers with different content and it would be great to have them both synced / connected to one kodi.

Thanks!

**Comment by croneter** [comment, marked as answer]
Never going to happen, sorry. It does not work conceptually - PKC could never tell whether you watched a certain movie on PMS 1 or PMS 2.

Use a dedicated Kodi profile per PMS instead, see https://github.com/croneter/PlexKodiConnect/wiki/Multiple-Plex-Media-Servers

**Reply by xinput** [reply]
Stupid me forgot to read the full guides.
Thanks for pointing me to this guide!
And also a big thanks for this great tool, really appreciate the work you‘ve done here!
